packages/rancher-webhook/package.yaml

```diff
@@ -1,2 +1,2 @@
-url: https://github.com/rancher/webhook/releases/download/v0.1.1/rancher-webhook-0.1.1.tgz
+url: https://github.com/rancher/webhook/releases/download/v0.1.2-rc1/rancher-webhook-0.1.2-rc1.tgz
```

Comment:
Are the only changes here the hardening stuff ? Want to make sure we aren't introducing anything else. I believe there was feature-flag code added.

Reply:
The changes include the hardening stuff and a pre-delete-hook to clean up `mutatingwebhookconfigurations`.

the feature flag is not backported.